tools/csv-merger/csv-merger.go

```diff
@@ -999,11 +999,12 @@ func getUXBackendServerDeployment() appsv1.DeploymentSpec {
 						Args: []string{"-provider=openshift",
 							"-https-address=:8888",
 							"-http-address=", "-email-domain=*",
-							"-upstream=https://localhost:8080/onboarding-tokens",
+							"-upstream=http://localhost:8080/onboarding-tokens",
 							"-tls-cert=/etc/tls/private/tls.crt",
 							"-tls-key=/etc/tls/private/tls.key",
 							"-cookie-secret-file=/etc/proxy/secrets/session_secret",
 							"-openshift-service-account=ux-backend-server",
+							"-openshift-delegate-urls={\"/\":{\"resource\":\"StorageCluster\",\"namespace\":\"openshift-storage\",\"verb\":\"get\"}}",
```

Comment:
you could use backticks (\`) rather than escaping quotes like, \`-openshift-....}\`